packages/graphql/src/mutations/_txHelper.js

```diff
@@ -746,11 +744,9 @@ export default function txHelper({
       hashCallbacks,
       receiptCallbacks,
       confirmCallbacks,
-      mutation
+      mutation,
+      reject
```

Comment:
Should maybe this be a try/catch around the `sendViaWeb3()` call instead of passing it as another argument?

Reply:
I tried that initially but for some reason the error wasn't bubbling up 🤷‍♂️